test/runtests.jl

```diff
@@ -39,8 +39,8 @@ datadir = joinpath(@__DIR__,"data")
     rm(fname)
   end
 
-  @testset "Legacy" begin
-    fname = joinpath(datadir,"legacy.gslib")
+  @testset "LegacyGrid" begin
+    fname = joinpath(datadir,"legacy_grid.gslib")
```

Comment:
I am not seeing the `legacy_grid.gslib` file in the PR? I can see the `legacy_pset.gsblib` which is great btw. Appreciate if you can run the tests locally before submitting a PR review request.

Reply:
That is strange. The file is in the commit 110129e. I always run the unit tests locally before submitting a PR review request.

Reply:
My mistake sorry. GitHub only shows the actual diffs, and file renames are only displayed with a tiny message in the PR, which I missed. I can see that you renamed legacy.gslib to legacy_grid.gslib 👍🏽